src/osu_put_latency.jl

```diff
@@ -0,0 +1,82 @@
+export OSUPutLatency
+
+struct OSUPutLatency <: MPIBenchmark
+    conf::Configuration
+    name::String
+end
+
+function OSUPutLatency(T::Type=Float32;
+                     filename::Union{String,Nothing}="julia_osu_put_latency.csv",
+                     kwargs...,
+                     )
+    return OSUPutLatency(
+        Configuration(T; filename, max_size=2 ^ 20, kwargs...),
+        "OSU Put Latency",
+    )
+end
+
+function osu_put_latency(T::Type, bufsize::Int, iters::Int, comm::MPI.Comm, synchronization_option::String)
+    win = MPI.Win_create(ones(T, bufsize), comm)
+    if(synchronization_option == "lock")
```

Comment:
[Don't parenthesize conditions](https://docs.julialang.org/en/v1/manual/style-guide/#Don't-parenthesize-conditions)
```suggestion
    if synchronization_option == "lock"
```